packages/@okta/vuepress-site/docs/release-notes/2023/index.md

```diff
@@ -0,0 +1,52 @@
+| [PBKDF2 Hashing Algorithm support is GA in Production](#pbkdf2-hashing-algorithm-support-is-ga-in-production) | December 9, 2022 |
+| [Revoke user sessions is GA in Production](#revoke-user-sessions-is-ga-in-production) | December 9, 2022 |
+| [Unusual telephony requests blocked by machine-learning measures](#unusual-telephony-requests-blocked-by-machine-learning-measures) | January 11, 2023 |
+| [Bugs fixed in 2023.01.0](#bugs-fixed-in-2023-01-0) | January 11, 2023 |
+
+#### Custom app login deprecated
+
+The custom app login feature is deprecated. This functionality is unchanged for orgs that actively use custom app login. Orgs that don't use custom app login should continue to use the [Okta-hosted sign-in experience](/docs/guides/redirect-authentication/) or [configure IdP routing rules](https://help.okta.com/okta_help.htm?type=oie&id=ext-cfg-routing-rules) that redirect users to the appropriate app to sign in. <!-- OKTA-564039-->
+
+#### Full regional support for AWS EventBridge Log Stream integrations is EA on Preview
+
+The Log Streaming API has expanded support for all commercial regions in the AWS EventBridge Log Stream integration. See [AWS EventBridge Setting property details](/docs/reference/api/log-streaming/#property-details-2). <!-- OKTA-540378-->
+
+#### Optional consent for OAuth 2.0 scopes is EA in Preview
+
+OAuth 2.0 Optional Consent provides an optional property that enables a user to opt in or out of an app's requested OAuth scopes. When optional is set to true for a scope, the user can skip consent for that scope. See Request user consent. <!-- OKTA-563884-->
+
+#### Password history policy enforced in strict mode is GA in Production
+
+When an admin [updates passwords](/docs/reference/api/users/#update-user) and sets the `strict` parameter to `true`, the [password history policy](/docs/reference/api/policy/#age-object) is now enforced.<!-- OKTA-563910-->
+
+#### PBKDF2 Hashing Algorithm support is GA in Production
+
+Okta now supports Password-Based Key Derivation Function 2 (PBKDF2) for hashing imported user passwords. This hashing algorithm is used to reduce the vulnerabilities of brute-force attacks. <!-- OKTA-563886-->
+
+#### Revoke user sessions is GA in Production
+
+You can end all Okta sessions for a user when resetting their password. All sessions of the specified user are revoked except for the current session. This option protects the user account from unauthorized access. See the `revokeSession` parameter in the [Users API](/docs/reference/api/users/#change-password). <!-- OKTA-542646-->
+
+#### Unusual telephony requests blocked by machine-learning measures
+
+SMS and voice requests are now blocked if an internal machine-learning-based toll fraud and abuse-detection model considers the requests unusual. Telephony requests that are blocked by the machine-learning model have a `DENY` status in the System Log. <!-- OKTA-562110-->
```

Comment:
> internal machine-learning-based toll fraud and abuse-detection model
This is a little hard to parse. Also, internal seems a little vauge.

What about something like"
> Okta uses an internal machine-learning-based fraud detection model to block unusual SMS or 
voice requests.

Reply:
the dev and PM were pretty adamant about the whole description being there. I did use the sentence structure that you suggest, but didn't remove "toll" or "abuse-detection" model.